tools/builder/react.js

```diff
@@ -52,7 +48,7 @@ gulp.task( 'react:master', function( done ) {
 				return;
 			}
 
-			buildStatic( done );
+			done();
```

Comment:
Prolly just `onBuild.bind( this, done )` would work here but good also as-is.

Reply:
3a2ec1bba